application/models/Template.php

```diff
@@ -578,10 +578,15 @@ public function deleteAssetVersion()
      * Return the standard template list
      * @return string[]
      * @throws Exception
+     * @deprecated Use surveyThemeHelper::getStandardTemplateList() instead.
      */
     public static function getStandardTemplateList()
     {
-        return array_keys(self::getTemplateInStandard());
+        // Refactored into surveyThemeHelper. Replaced the code here
+        // by a call to the helper to avoid code duplication while keeping
+        // backwards compatibility.
+        Yii::import('application.helpers.surveyThemeHelper');
```

Comment:
You know, you can put the file as a service class and have autoload load the file instead of manually import it everywhere. :)

Reply:
Or maybe not, since this is LS3...